test/extended/util/framework.go

```diff
@@ -1949,6 +1949,17 @@ func GetRouterPodTemplate(oc *CLI) (*corev1.PodTemplateSpec, string, error) {
 }
 
 func FindRouterImage(oc *CLI) (string, error) {
+	exists, err := DoesApiResourceExist(oc, "clusteroperators")
+	if err != nil {
+		return "", err
+	}
+	if !exists {
+		deployment, err := oc.AdminKubeClient().AppsV1().Deployments("openshift-ingress").Get(context.Background(), "router-default", metav1.GetOptions{})
```

Comment:
In 4.12 taking `deployment.Spec.Template.Spec.Containers[0].Image` from the deployment gets me `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a955bf1686fcb4f11cd23880be6025c2dc75252d318931211554be86f4f5d0f1`. Taking `.Status.Versions[*].Version` from clusteroperator/ingress I get `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a955bf1686fcb4f11cd23880be6025c2dc75252d318931211554be86f4f5d0f1`. @pacevedom is it always guaranteed the images will be identical?

Reply:
@openshift/networking can you confirm accessing `deployment.Spec.Template.Spec.Containers[0].Image` of `router-default` deployment gets the same image as from the `ingress` clusteroperator object?

Reply:
Yes, `deployment.Spec.Template.Spec.Containers[0].Image` of the `openshift-ingress/router-default` deployment is the same image as the "ingress-controller" image in the `ingress` clusteroperator, on both MicroShift and standard OpenShift.  This approach looks reasonable.

Reply:
This is output from a 4.12 cluster:
```
$ oc get deployment -n openshift-ingress router-default -o jsonpath='{.spec.template.spec.containers[*].image}'
quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:72c751aa148bf498839e6f37b304e3265f85af1e00578e637332a13ed9545ece

$ oc get clusteroperator ingress -o jsonpath='{.status.versions[?(@.name=="ingress-controller")].version}'
quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:72c751aa148bf498839e6f37b304e3265f85af1e00578e637332a13ed9545ece
```

`router-default` is MicroShift equivalent for `ingress-controller` pod in OCP.